lang/c++/impl/NodeImpl.cc

```diff
@@ -519,6 +519,16 @@ void NodeArray::printJson(std::ostream &os, size_t depth) const {
         os << indent(depth + 1) << R"("doc": ")"
            << escape(getDoc()) << "\",\n";
     }
+    if (logicalType().type() != LogicalType::NONE) {
+        os << indent(depth);
+        logicalType().printJson(os);
+        os << ",\n";
+    }
+    if (logicalType().type() != LogicalType::NONE) {
+        os << indent(depth);
+        logicalType().printJson(os);
+        os << ",\n";
+    }
```

Comment:
is it intentional that this block is duplicated with the one above?

Reply:
No, I think that was just a copy/paste error or a merge error when I was rebasing things.